### PR TITLE
chore(collaboration): update collaboration logic and messages

### PR DESCRIPTION
**Pull request by KennethBWSong**
List collaborator messages:
image

### CODE REVIEW

**Review by xzf0587 — DISMISSED** [review]

**Review by SLdragon — APPROVED** [review]

File: packages/fx-core/resource/package.nls.json [hunk 1]
```diff
@@ -157,8 +157,7 @@
   "core.handleConfigFile.FailedToUpdateAzureParameters": "Failed to update Azure parameters for %s environment.",
   "core.appSettingsUtil.FailedToResetAppSettingsDevelopment": "Failed to reset values in appsettings.Development.json.",
   "core.backupFiles.FailedToBackupFiles": "Failed to backup %s.",
-  "core.collaboration.ListCollaboratorsSuccess": "List Microsoft 365 Teams App%s owners success, you can view it in [output channel](%s).",
-  "core.collaboration.WithAadApp": "(with AAD App)",
+  "core.collaboration.ListCollaboratorsSuccess": "List Microsoft 365 App owners success, you can view it in [output channel](%s).",
```
Comment (sffamily) [inline packages/fx-core/resource/package.nls.json:160]: success -> succeeded

Comment (sffamily) [reply packages/fx-core/resource/package.nls.json:160]: quote :

'List Microsoft 365 App owners' succeeded,

File: packages/fx-core/resource/package.nls.json [hunk 2]
```diff
@@ -172,10 +171,12 @@
   "core.collaboration.AzureTip": "\nIf added user cannot access Azure resources, you need to manually setup access policy via Azure portal, refer to this link for more details: ",
   "core.collaboration.ListingM365Permission": "Listing Microsoft 365 permissions\n",
   "core.collaboration.AccountUsedToCheck": "Account used to check: ",
-  "core.collaboration.StartingListAllTeamsAppOwners": "Starting list all teams app owners for environment: ",
+  "core.collaboration.StartingListAllTeamsAppOwners": "\nStarting list all teams app owners for environment: ",
```
Comment (sffamily) [inline packages/fx-core/resource/package.nls.json:174]: Starting to list

File: packages/fx-core/resource/package.nls.json [hunk 3]
```diff
@@ -172,10 +171,12 @@
...
+  "core.collaboration.StartingListAllAadAppOwners": "\nStarting list all Azure Active Directory app owners for environment: ",
```
Comment (sffamily) [inline packages/fx-core/resource/package.nls.json:175]: to list

File: packages/fx-core/resource/package.nls.json [hunk 4]
```diff
@@ -172,10 +171,12 @@
...
   "core.collaboration.M365TeamsAppId": "Microsoft 365 Teams App (ID: ",
-  "core.collaboration.SsoAadAppId": "), SSO AAD App (ID: ",
+  "core.collaboration.SsoAadAppId": "SSO AAD App (ID: ",
```
Comment (sffamily) [inline packages/fx-core/resource/package.nls.json:177]: is it a typo for "("?

Comment (KennethBWSong) [reply packages/fx-core/resource/package.nls.json:177]: “(” is expected here. We are outputting: 'SSO AAD App (ID: ' + aadAppId + ')'.

File: packages/fx-core/resource/package.nls.json (hunk 2 above)
Comment (KennethBWSong) [reply packages/fx-core/resource/package.nls.json]: SSO AAD App (ID:

File: packages/fx-core/resource/package.nls.json (hunk 1 above)
Comment (KennethBWSong) [reply packages/fx-core/resource/package.nls.json]: Updated, thx!

**Review by sffamily — APPROVED** [review]

**Review by xzf0587 — APPROVED** [review]

**Pull request merged**